crypto/key_manager.go

```diff
@@ -25,7 +25,7 @@ import (
 	"io/ioutil"
 
 	"github.com/golang/glog"
-	"github.com/google/trillian"
+	spb "github.com/google/trillian/proto/signature"
```

Comment:
As we've done with all ours except the top level one. What did you think about moving it inside the relevant package and calling it somethingpb?